documentation/environments/node.md

```diff
@@ -19,7 +19,7 @@ The following plugins are included:
 - transform-class-properties
 - transform-flow-strip-types
 
-### 6.x Babel Plugins
```

Comment:
Can you also update the paragraph under the "JavaScript" heading?

> Node 0.10.33 and 6.0 are currently supported, each with the option of transpiling using Babel JS. Node 6.0 without babel is the recommended option due to it supporting most ES2015 features natively, and because it does not incur the additional transpilation performance penalty that Babel JS causes.